client/codegen_pinvoke/Main.cs

```diff
@@ -91,16 +92,20 @@ static int Main(string[] args)
                     lib.output_dir.Delete(true);
                 }
 
+                // TODO: move this string somewhere more sensible

```

Comment:
This is fine for a location for the string. Executables will call the `Codegen` class with the correct string, which is what is happening here.

```suggestion
```